src/traces/scattersmith/format_labels.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var Lib = require('../../lib');
+var Axes = require('../../plots/cartesian/axes');
+
+module.exports = function formatLabels(cdi, trace, fullLayout) {
+    var labels = {};
+
+    var subplot = fullLayout[trace.subplot]._subplot;
+    var radialAxis;
+    var angularAxis;
+
+    // for scatterpolargl texttemplate, _subplot is NOT defined, this takes part during the convert step
+    // TODO we should consider moving the texttemplate formatting logic to the plot step
+    if(!subplot) {
+        subplot = fullLayout[trace.subplot];
+        radialAxis = subplot.realaxis;
+        angularAxis = subplot.imaginaryaxis;
+    } else {
+        radialAxis = subplot.radialAxis;
+        angularAxis = subplot.angularAxis;
+    }
+
+    var rVal = radialAxis.c2l(cdi.re);
+    labels.rLabel = Axes.tickText(radialAxis, rVal, true).text;
+
+    // N.B here the ° sign is part of the formatted value for thetaunit:'degrees'
+    var thetaVal = angularAxis.thetaunit === 'degrees' ? Lib.rad2deg(cdi.im) : cdi.im;
+    labels.thetaLabel = Axes.tickText(angularAxis, thetaVal, true).text;
+
+    // labels.reLabel = Axes.tickText(radialAxis, cdi.re, true).text;
+    // labels.imLabel = Axes.tickText(radialAxis, cdi.im, true).text;
```

Comment:
We could :hocho: these lines. No?

Reply:
fdf7776b3ace64a4605f724d28b966a4b6a67a42